zookeeper-docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -265,18 +266,25 @@ server.3=zoo3:2888:3888</programlisting>
           role="bold">server.id=host:port:port</emphasis>. The parameters <emphasis
           role="bold">host</emphasis> and <emphasis
           role="bold">port</emphasis> are straightforward. You attribute the
-          server id to each machine by creating a file named
+          server id to each machine by setting <emphasis role="bold">server.id</emphasis>
+          to a unique integer for each zookeeper server.To keep backwards compatibility,
+          you can still creat a file named
```

Comment:
nit - spelling 'create'

Reply:
we should also talk about the importance of not changing the id since it becomes much easier with the configuration file option. we should have documented that anyway, but it becomes much more important with this change since the id is decoupled from the data.